### DISCUSSION TITLE
FluentUI template and dotnet 9

### DISCUSSION THREAD

**Discussion by prod42net** [microsoft/fluentui-blazor#3028] [Q&A]
With fluentui 4.10.4 and higher dotnet 9 should be supported. For some reason the template selection within rider is grayed out if I select dotnet 9. I can start with dotnet 8 and the fluentui template and do upgrade to dotent 9 though.
Is there any way to enable the template within the new solution dialog? Anyone else with the same issue or is it just my setup? I am using the full licensed Rider version on a mac.

**Comment by vnbaaij** [comment]
We worked on the templates and SDK support for v4.11.

Can you try to install the beta templates package and try with that one?

**Reply by prod42net** [reply]
That works. Thanks.
